lib/utils/parser.rb

```diff
@@ -92,6 +104,12 @@ def parse_mount_options(mount_line, compatibility = false)
 
     mount_options
   end
+
+  # Device-/Sharename or Mountpoint includes whitespaces?
+  def includes_whitespaces?(mount_line)
+    ws = mount_line.match(/^(.+)\son\s(.+)\stype\s.*$/)
+    ws.captures[0].include? ' ' or ws.captures[1].include? ' '
```

Comment:
Same here.. each of the `include?` method calls should wrap their arguments with parentheses, please. :)